.github/workflows/release-chart.yml

```diff
@@ -0,0 +1,48 @@
+name: Release newrelic infrastructure chart
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+#   Sometimes chart-releaser might fetch an outdated index.yaml from gh-pages, causing a WAW hazard on the repo
+#   This job checks the remote file is up to date with the local one on release
+  validate-gh-pages-index:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: gh-pages
+      - name: Download remote index file and check equality
+        run: |
+          curl -vsSL https://newrelic.github.io/nri-kubernetes/index.yaml > index.yaml.remote
+          LOCAL="$(md5sum < index.yaml)"
+          REMOTE="$(md5sum < index.yaml.remote)"
+          echo "$LOCAL" = "$REMOTE"
+          test "$LOCAL" = "$REMOTE"
```

Comment:
I'd like to pint that this step can fail if there is no `index.yaml` created in the `gh-pages` branch.

But I swa you put [an index.yaml in that branch](https://github.com/newrelic/nri-kubernetes/commit/1ae507d0c2572740219614886f53ca28948c3289). Are we completely sure that this is the bare minimum for `cr` to work? Because I put a completely empty `index.yaml` (`touch index.yaml && git add -A && git push`) and it failed.

I would change my PR on the metrics adapter if you confirm me that this works :)

Reply:
I run 
```
git checkout --orphan gh-pages
 - Deleted everything but the gitignore
helm repo index .
```

In the forked it worked!

Reply:
Really cool. I did not think about using `helm` to create an empty index. Now I feel dumb 😅